# Robot name goes here
fire_fight_1: # change the name
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  rear_wheelR_velocity_controller:
    type: velocity_controllers/JointVelocityController 
    joint: W4
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
  # Controller 2
  rear_wheelL_velocity_controller:
    type: velocity_controllers/JointVelocityController 
    joint: W3
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
  # Controller 3
  # front_wheelL_effort_controller:
  #   type: effort_controllers/JointEffortController 
  #   joint: W1
  #   pid : {p: 100.0 , i: 1.0 , d: 1.0} 
  # # Controller 4
  # front_wheelR_effort_controller:
  #   type: effort_controllers/JointEffortController 
  #   joint: W2
  #   pid : {p: 100.0 , i: 1.0 , d: 1.0} 
  # Controller 5
  front_steerL_position_controller:
    type: position_controllers/JointPositionController 
    joint: H1
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
  # Controller 6
  front_steerR_position_controller:
    type: position_controllers/JointPositionController 
    joint: H2
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
  # Controller 7
  first_arm_controller:
    type: position_controllers/JointPositionController 
    joint: A1
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
    
  second_arm_controller:
    type: position_controllers/JointPositionController 
    joint: A2
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
    
  third_arm_controller:
    type: position_controllers/JointPositionController 
    joint: A3
    pid : {p: 100.0 , i: 1.0 , d: 1.0} 
 